src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/Marshalling/CustomTypeMarshallingGenerator.cs

```diff
@@ -105,6 +106,11 @@ public IEnumerable<StatementSyntax> Generate(TypePositionInfo info, StubCodeCont
             return Array.Empty<StatementSyntax>();
         }
 
+        private bool GenerateByValueOutMarshalling(TypePositionInfo info)
```

Comment:
nit: maybe rename this to `ShouldGenerateByValueOutMarshalling` or something like that since it returns a bool and doesn't do the generating?